scout/server/blueprints/clinvar/templates/multistep_add_variant.html

```diff
@@ -375,12 +374,25 @@ <h3 class="fs-subtitle">Information provided by filling in these fields will be
 	});
 });
 
+var conditionsPlaceHolders = {"OMIM": "e.g. 100800, ..", "HPO": "e.g. HP:0002839, ..", "MedGen": "e.g. C0001080, ..", "MeSh": "e.g. D000130, ..", "MONDO": "e.g. MONDO:0007947, ..", "Orphanet": "e.g. ORPHA155, .."};
+
 // Populate condition list
 $('#condition_tags').select2({
   tags: true,
   theme: 'bootstrap-5',
   tokenSeparators: [','],
-  placeholder: "e.g. OMIM:604233,OMIM:300491,..",
+  allowClear: true
+});
+
+// reset and modify conditions field's placeholder when condition ID type changes
+$(function(){
+  $("#condition_type").change(function(){
+    $('#condition_tags').val(null).trigger('change');
+    selectedCondId = $("#condition_type option:selected").text();
+    $('#condition_tags').select2({
+      placeholder: conditionsPlaceHolders[selectedCondId]
+    });
+  });
```

Comment:
Now all that is missing is an update of the actual choices? 😉 
Its fine without as well: if one has OMIM terms, it is more than reasonable to use them by default rather than HPO terms for like for an unknown disorder. That is presumably kind of why ClinVar only accept one kind anyway?

But as the placeholders look like one can swap, I can imagine someone being disappointed they can't just switch back and forth between OMIM and HPO:

![Screenshot 2022-12-05 at 13 36 47](https://user-images.githubusercontent.com/758570/205639894-23986554-f879-42ed-a555-a8e1befe575d.png)

Reply:
I've been try to fix it as you suggest, but still doesn't work. Changing values of the select2 lib elements is not so straightforward as modifying a select. I'll try again tomorrow.

Reply:
Ok, now it should work as intended!